hadoop-project-dist/pom.xml

```diff
@@ -106,7 +106,7 @@
           <source>${maven.compile.source}</source>
           <charset>${maven.compile.encoding}</charset>
           <reportOutputDirectory>${project.build.directory}/site</reportOutputDirectory>
-          <destDir>${project.build.directory}/api</destDir>
+          <outputDirectory>${project.build.directory}/api</outputDirectory>
```

Comment:
here both are defined
https://maven.apache.org/plugins/maven-javadoc-plugin/javadoc-mojo.html

Is there any behaviour change here? Both have different default values as well

cc. @GauthamBanasandra I think you were around this area, does it look good to you?

Reply:
The previous `destDir` was used for API directory and was the alias of `outputDirectory`.
![Screenshot 2023-06-05 at 10 58 08 PM](https://github.com/apache/hadoop/assets/9700541/d71657f0-cf07-4c56-8f14-1199a12150d4)

Reply:
Shall we skip this line simply?

Reply:
I would yes, lets not do it, similar change is there in [pom.xml](https://github.com/apache/hadoop/pull/5592/files#diff-9c5fb3d1b7e3b0f54bc5c4182965c4fe1f9023d449017cece3005d3f90e8e4d8)

So, we can go ahead without this destDir -> outputDirectory change